src/plugins/intel_gpu/src/kernel_selector/cl_kernels/swiglu_gpu_opt.cl

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// From A&S Handbook of Mathematical Functions with Formulas, Graphs, and Mathematical Tables
+// Formula section 7.1.26
+
+ACCUMULATOR_TYPE fast_erf(ACCUMULATOR_TYPE x) {
+    const ACCUMULATOR_TYPE a1 = 0.254829592f;
```

Comment:
Should these coefficients be of `ACCUMULATOR_TYPE`? Integer types can be used as accumulators too (`GetAccumulatorType()` function allows them)

Reply:
We are dealing with ACCUMULATOR_TYPE inputs so I guess it also makes sense that the coefficients would be of ACCUMULATOR_TYPE, unless I am missing something